axelor-supplychain/src/main/resources/views/Pack.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<object-views xmlns="http://axelor.com/xml/ns/object-views"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://axelor.com/xml/ns/object-views http://axelor.com/xml/ns/object-views/object-views_5.1.xsd">
+  
+    <action-view id="supplychain-action-wizard-view-add-pack" name="action-wizard-view-add-pack" title="Add Pack" model="com.axelor.apps.base.db.Wizard">
+      <view type="form" name="add-pack-wizard"/>
+      <view-param name="popup" value="reload"/>
+      <view-param name="show-toolbar" value="false"/>
+      <view-param name="show-confirm" value="false" />
+      <view-param name="popup-save" value="false"/>
+      <context name="_showRecord" expr="eval: _id"/>
+      <context name="model" expr="eval: _model"/>
+    </action-view>
+  
+    <form id="supplychain-add-pack-wizard" name="add-pack-wizard" model="com.axelor.apps.base.db.Wizard" title="Add Pack" onNew="action-sale-order-add-pack-attrs-onload" extension="true">
```

Comment:
Rename "action-sale-order-add-pack-attrs-onload" to "action-add-pack-wizard-attrs-onload"